code/game/objects/structures/bedsheet_bin.dm

```diff
@@ -222,6 +222,16 @@ LINEN BINS
 	item_color = "ian"
 	dream_messages = list("a dog", "a corgi", "woof", "bark", "arf")
 
+/obj/item/bedsheet/cosmos
+	name = "cosmic space blankie"
+	desc = "Made of the dreams of space children everywhere."

```

Comment:
Not something I wouldn't merge this over, but I think there's potential for a way cooler, more cosmic desc. I mean, look at that sprite. It's so beautiful...